### ISSUE TITLE
Sea Pup not working: Brightness controls?

### ISSUE THREAD

**Issue by gtoal** [amaiorano/vectrexy#7]
Running the game Sea Pup (the "No PB6" version of the binary from https://portauthority.itch.io/sea-pup ) is unplayable on Vectrexy - although I think the only issue may be the programmer is drawing with some vector intensities set too low. However there is no "brightness knob" setting that would let the user compensate. (That I could find. Apologies if I missed something?) Give it a try and see for yourself.

**Comment by amaiorano** [comment]
Thanks, Graham, I'll check it out as soon as I can. I assume this game works fine in other emulators?

**Comment by gtoal** [comment]
> Thanks, Graham, I'll check it out as soon as I can. I assume this game works fine in other emulators?

The only other one I have installed at the moment is Roger's "Vectrex Online" which works OK.

**Comment by amaiorano** [comment]
Thanks for reporting this. I spent some time investigating, and figured out the root of the issue. This is the first game I test (I think) that uses the shift register differently. Most games use the "Shift out under 02" mode, while this game uses that *and* also disables the shift register at times to manually control blanking using the peripheral control register. I implemented this (it was partly done already), and now all looks good.

The next build should include this fix.
